atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/transaction/impl/TestTransactionManagerImpl.java

```diff
@@ -157,7 +157,7 @@ public TestTransactionManagerImpl(
                 AbstractTransactionTest.GET_RANGES_THREAD_POOL_SIZE,
                 AbstractTransactionTest.DEFAULT_GET_RANGES_CONCURRENCY,
                 sweepQueue,
-                deleteExecutor,
+                new DefaultDeleteExecutor(createAssertKeyValue(keyValueService, lockService), deleteExecutor),
```

Comment:
This is not great, but tests still pass. It's inside a super constructor invocation, there's no way to pull out the right things here.